### PR TITLE
Update to RAS in CCPP, update submodule pointer for ccpp-framework (CI tests)

### LINKED ISSUE #235: Removed unused/old 3D diagnostic tendencies from GFS_typedefs.F90 and CCPP-physics GFS_DCNV_generic ?
The following three variables are passed to CCPP's `GFS_DCNV_generic.{F90,meta}`, but are no longer used (set) in the scheme. We need to decide what to do with the code and this variables. If the code should be activated again inside the `ldiag3d = .true.` block, then the allocation in `GFS_typedefs.F90` must be activated, too. Otherwise remove those three variables from both the scheme and `GFS_typedefs`.
```
[upd_mf]
  standard_name = cumulative_atmosphere_updraft_convective_mass_flux
  long_name = cumulative updraft mass flux
  units = kg m-1 s-2
  dimensions = (horizontal_loop_extent,vertical_dimension)
  type = real
  kind = kind_phys
[dwn_mf]
  standard_name = cumulative_atmosphere_downdraft_convective_mass_flux
  long_name = cumulative downdraft mass flux
  units = kg m-1 s-2 
  dimensions = (horizontal_loop_extent,vertical_dimension)
  type = real
  kind = kind_phys
[det_mf]
  standard_name = cumulative_atmosphere_detrainment_convective_mass_flux
  long_name = cumulative detrainment mass flux
  units = kg m-1 s-2
  dimensions = (horizontal_loop_extent,vertical_dimension)
  type = real
  kind = kind_phys
```

### LINKED ISSUE #476: fix the argument list in inline post stub subroutine
## Description
Provide a clear and concise description of what the bug is.
It is found that the model can't compile with "NO_INLINE_POST". The argument variable fldbundle in subroutine inline_post_getattr in inline_post_stub.F90 needs to be removed.

### To Reproduce:
What compilers/machines are you seeing this with?
Check out the latest develop branch and compile the model with -DNO_INLINE_POST". An error message will show up at compile time:
/lfs/h1/emc/nceplibs/noscrub/gwv/u/ufs-weather-model/FV3/io/module_wrt_grid_comp.F90(1169): error #6631: A non-optional actual argument must be present when invoking a procedure with an explicit interface.   [FLDBUNDLE]
        call inline_post_getattr(wrt_int_state)

## Additional context
An FV3 PR will be created.
- needed for noaa-emc/fv3atm/pull/<pr_number>

### PR DESCRIPTION
**Pull request by climbfuji**
## Description

This PR does the following:
- update `.gitmodules` and submodule pointer for ccpp-physics for the changes described in https://github.com/NCAR/ccpp-physics/pull/58
- update submodule pointer for ccpp-framework for recent changes merged into the repository (no code changes, only Travis CI tests of the framework code)
- reactivate the 3d mass flux diagnostics to solve issue https://github.com/NOAA-EMC/fv3atm/issues/235
- bugfix for compiling model without inline post (from @junwang-noaa)
 
### Issue(s) addressed

Fixes https://github.com/NOAA-EMC/fv3atm/issues/235
Fixes https://github.com/ufs-community/ufs-weather-model/issues/476
## Testing

See https://github.com/ufs-community/ufs-weather-model/pull/448.

## Dependencies

https://github.com/NCAR/ccpp-physics/pull/585
https://github.com/NOAA-EMC/fv3atm/pull/252
https://github.com/ufs-community/ufs-weather-model/pull/448

### CODE REVIEW

**Comment by SMoorthi-emc** [comment]
we need to add 
"    ! -- setup aerosol scavenging factors
    n = max(Model%ntrac, Model%ntchm)
    allocate(Model%fscav(n))
    Model%fscav = -9999.0"
replacing 
    "allocate(Model%fscav(Model%ntchm))"
in GFS_typedef.F90

File: ccpp/suites/suite_FV3_GFS_v15p2_ras.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_v15p2_ras" version="1">
```
Comment (junwang-noaa) [inline ccpp/suites/suite_FV3_GFS_v15p2_ras.xml]: Is it possible to create a suite_FV3_GFS_v16_ras.xml suite file? The operational GFSv16 will be implemented this week, and we hope to update global suite files to be GFS_v16 based.

**Comment by SMoorthi-emc** [comment]
It should be simple to do; just take the suite with sas and replace
"       <scheme>samfdeepcnv</scheme>"
with
"       <scheme>rascnv</scheme>"
and change the suite name.

**Comment by climbfuji** [comment]
> we need to add
> " ! -- setup aerosol scavenging factors
> n = max(Model%ntrac, Model%ntchm)
> allocate(Model%fscav(n))
> Model%fscav = -9999.0"
> replacing
> "allocate(Model%fscav(Model%ntchm))"
> in GFS_typedef.F90

Done.

**Review by junwang-noaa — APPROVED** [review]

File: ccpp/suites/suite_FV3_GFS_v15p2_ras.xml (hunk 1 above)
Comment (climbfuji) [reply ccpp/suites/suite_FV3_GFS_v15p2_ras.xml]: I replaced `ccpp/suites/suite_FV3_GFS_v15p2_ras.xml` with `ccpp/suites/suite_FV3_GFS_v16_ras.xml` and also changed the regression tests in the ufs-weather-model PR.

**Review by SMoorthi-emc — APPROVED** [review]

**Comment by climbfuji** [comment]
Ready to merge.

**Pull request merged**